test/integration/rt_channel_test.exs

```diff
@@ -885,19 +885,11 @@ defmodule Realtime.Integration.RtChannelTest do
 
       assert_receive %Phoenix.Socket.Message{event: "phx_reply"}, 500
       assert_receive %Phoenix.Socket.Message{event: "presence_state"}, 500
+
       tenant = Tenants.get_tenant_by_external_id(@external_id)
       Realtime.Api.update_tenant(tenant, %{jwt_jwks: %{keys: ["potato"]}})
-
-      assert_receive %Phoenix.Socket.Message{
-                       topic: ^realtime_topic,
-                       event: "system",
-                       payload: %{
-                         "extension" => "system",
-                         "message" => "Server requested disconnect",
```

Comment:
This seemed like a good message, because we can have them console log something to see why they were disconnected.

If we just disconnect they will think it's an error on our end.

Reply:
Yes but with this approach of killing the socket we can't send it as the socket will close before we can send the message from the RT channel 😞 

I'm thinking of keeping implementation as it was and then from RealtimeChannel emit the kill socket message... more steps but we can get some of this "niceties"